dotcom-rendering/src/web/components/FilterKeyEventsToggle.importable.tsx

```diff
@@ -22,9 +22,10 @@ const toggleWrapperStyles = css`
 
 interface Props {
 	filterKeyEvents: boolean;
+	id: string;
```

Comment:
Thinking about it, maybe `id` isn't the right name for this prop? What it actually signifies is the breakpoint (roughly at least). 

With that in mind, one approach could be to call the prop `breakpoint` and limit the values we can pass to it to `desktop` and `mobile`.

That would then look like this:
```
interface Props {
   filterKeyEvents: boolean;
   breakpoint: "mobile" | "desktop"
}
```

Does that make sense? We'd need to update the other places we've used `id` to be `breakpoint` as well, but again I think it explains what's going on better.

Reply:
@silvacb I'm free this morning so if you're not hacking I'm very happy to go through all these comments together :) (I'd recommend getting involved in the hackday though!)